app/models/email_template.rb

```diff
@@ -42,6 +43,8 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :daily_summary
+      User
```

Comment:
Okk its great then. It is easier to implement for me using simple mailers. Ill complete using by creating mailers :)